tensorflow/python/feature_column/feature_column_v2_test.py

```diff
@@ -1837,6 +1837,22 @@ def test_sparse_combiner(self):
         sess.run(bias.assign([5.]))
         self.assertAllClose([[1005.], [5010.]], self.evaluate(predictions))
 
+  def test_sparse_combiner_sqrtn(self):
+    wire_cast = fc.categorical_column_with_hash_bucket('wire_cast', 4)
+    with ops.Graph().as_default():
+      wire_tensor = sparse_tensor.SparseTensor(
+          values=['omar', 'stringer', 'marlo'],  # hashed to = [2, 0, 3]
+          indices=[[0, 0], [1, 0], [1, 1]],
+          dense_shape=[2, 2])
+      features = {'wire_cast': wire_tensor}
+      model = fc.LinearModel([wire_cast], sparse_combiner='sqrtn')
+      predictions = model(features)
+      wire_cast_var, bias = model.variables
+      with _initialized_session() as sess:
+        sess.run(wire_cast_var.assign([[10.], [100.], [1000.], [10000.]]))
```

Comment:
Lets future proof this a little bit more. Can we make this self.evaluate(wire_cast_var.assign....)

Similarly below.

Reply:
@rohan100jain thanks for the review, i have updated the PR as per your comments, kindly check and approve.